timeseries/tests/unittests/test_ts_dataset.py

```diff
@@ -1011,3 +1011,10 @@ def test_when_timestamps_have_datetime64_type_then_tsdf_can_be_constructed(dtype
     df[TIMESTAMP] = df[TIMESTAMP].astype(dtype)
     assert df[TIMESTAMP].dtype == dtype
     TimeSeriesDataFrame.from_data_frame(df)
+
+
+def test_when_to_data_frame_called_then_return_values_is_a_pandas_df():
+    tsdf = SAMPLE_TS_DATAFRAME.copy()
+    df = tsdf.to_data_frame()
```

Comment:
De we need to test that `df` can be directly converted to `TimeSeriesDataFrame`?
With or without resetting the index?